dmsite/frontend/src/SearchResultsPage/SearchResultsCell.jsx

```diff
@@ -12,7 +12,8 @@ class SearchResultsCell extends React.Component {
     // TODO: modify the onClick to route the the appropriate link
     return (
       <ListGroupItem
-        action onClick={() => { this.props.history.push('/'); }}
+      /* future bug, can't redirect because home has multiple campaigns */
+        action onClick={() => { this.props.history.push('/home'); }}
```

Comment:
Instead of pushing to home, you could push to campaign list - it'll be a bit awkward but it would make the system more smooth.

this.props.history.push('/campaign')